src/Bookings.js

```diff
@@ -1,18 +1,39 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
 import Search from "./Search.js";
-// import SearchResults from "./SearchResults.js";
+import SearchResults from "./SearchResults.js";
 // import FakeBookings from "./data/fakeBookings.json";
 
 const Bookings = () => {
+  const [bookings, setBookings] = useState([]);
+
+  useEffect(() => {
+    fetch(`https://cyf-react.glitch.me`)
+      .then(res => res.json())
+      .then(data => {
+        setBookings(data);
+      });
+  }, []);
+
   const search = searchVal => {
-    console.info("TO DO!", searchVal);
+    let searchedBookings = bookings.filter(booking => {
+      if (
+        booking.firstName.toLowerCase() === searchVal.toLowerCase() ||
+        booking.surname.toLowerCase() === searchVal.toLowerCase()
+      ) {
+        return true;
+      } else {
+        return false;
+      }
+    });
```

Comment:
You don't really need the 'if statement ' as the 'filter' method implicitly use it :
```suggestion
    let searchedBookings = bookings.filter(booking => {
      return (
        booking.firstName.toLowerCase() === searchVal.toLowerCase() ||
        booking.surname.toLowerCase() === searchVal.toLowerCase()
      )     });
```